php-83/Dockerfile

```diff
@@ -133,10 +133,42 @@ RUN make -j1 install_sw install_ssldirs
 RUN mkdir -p ${INSTALL_DIR}/bref/ssl && curl -Lk -o ${CA_BUNDLE} ${CA_BUNDLE_SOURCE}
 
 
+###############################################################################
+# ICU
+# We need to compile ICU from source because Amazon Linux 2 only has ICU 50.2
+# which is cripplingly old and we want to support newer Unicode standards
```

Comment:
do we know which minimum version we want or precisely which features (which will give us the version)?

I'm asking for #122

Reply:
The very latest version has the very latest unicode standard. We're not really looking for new "features", more so the implementation of the characters from the latest standard.

Reply:
https://cldr.unicode.org/downloads/cldr-47 is the standard implemented by 77.1.